cli/flags.rs

```diff
@@ -1760,12 +1763,13 @@ fn permission_args(app: Command) -> Command {
     )
     .arg(Arg::new("prompt").long("prompt").hide(true).help(
       "deprecated: Fallback to prompt if required permission wasn't passed",
-    ))
-    .arg(
-      Arg::new("no-prompt")
-        .long("no-prompt")
-        .help("Always throw if required permission wasn't passed"),
-    )
+    ));
+  let no_prompt_arg = Arg::new("no-prompt")
+    .long("no-prompt")
+    .help("Always throw if required permission wasn't passed");
+  #[cfg(not(test))]
```

Comment:
Why `cfg(not(test)` ?

Reply:
So that tests of `flags` module do not depend on the environmental variables - see https://github.com/denoland/deno/pull/14209#discussion_r846864313

Reply:
So the unit tests in this module aren't affected by the env var for local dev